tests/foreman/api/test_registration.py

```diff
@@ -445,3 +446,66 @@ def test_positive_katello_ca_crt_refresh(
     # check if the certificate file is refreshed
     ca_file_after_refresh = len(str(rhel_contenthost.execute(f'cat {katello_ca_crt_path}')))
     assert ca_cert_file == ca_file_after_refresh
+
+
+@pytest.mark.rhel_ver_match('[^7]')
```

Comment:
is it really required to run for all rhel versions? I feel default should be suffice to test